vite.config.ts

```diff
@@ -71,6 +71,7 @@ export default defineConfig({
     BUILD_FOSS_ONLY: !!process.env.BUILD_FOSS_ONLY,
   },
   test: {
+    exclude: ["e2e/", "node_modules/"],
```

Comment:
This should fix `pnpm test`:
```suggestion
    exclude: ["**/e2e/**", "**/node_modules/**"],
```